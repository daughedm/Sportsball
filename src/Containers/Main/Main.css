.orange-container {
  width: 70%;
  height: 100vh;
  background: radial-gradient(circle, #F26F44 0%, #A65638 100%);
  /* float: right; */
  padding: 60px 80px 60px 80px;
  display: inline-block;
}

.left-container {
  width: 30%;
  height: 100vh;
  padding: 50px;
  display: inline-block;
  vertical-align: top;
  background-image: url('../../Assets/images/horizBG.jpg');
  background-repeat: no-repeat;
  background-size: cover;
}

.talking-point {
  font-size: 20px;
  color: #F4F1E7;
  border-bottom: 1px solid #F4F1E7;
  padding-bottom: 25px;
  letter-spacing: 1px;
}

.light {
  font-weight: 400;
}

.talking-points-container {
  width: 100%;
  height: 100%;
}

.u-list {
  padding: 0;
  overflow-x: hidden;
  max-height: 90%;
}

.point {
  overflow: hidden;
  font-family: 'Noto Serif', serif;
  font-size: 50pt;
  font-weight: 700;
  color: #F4F1E7;
  padding: 35px 0 35px 0;
  margin-right: 40px;
}

.border-bottom {
  border-bottom: 1px solid #F4F1E7;
}
 
.sportsball-logo-side {
  width: 100%;
  display: block;
}

.container {
  margin: 0;
  padding: 0;
}

.team-selected {
  color: #F26F44;
  font-family: 'Noto Serif', serif;
  font-size: 40px;
  text-align: center;
  letter-spacing: 2px;
  margin-top: 50px;
}

.animated {
  -webkit-animation-duration: 1s;
  animation-duration: 1s;
  -webkit-animation-fill-mode: both;
  animation-fill-mode: both;
}

@-webkit-keyframes bounceInRight {
  from,
  60%,
  75%,
  90%,
  to {
    -webkit-animation-timing-function: cubic-bezier(0.215, 0.61, 0.355, 1);
    animation-timing-function: cubic-bezier(0.215, 0.61, 0.355, 1);
  }
  from {
    opacity: 0;
    -webkit-transform: translate3d(3000px, 0, 0);
    transform: translate3d(3000px, 0, 0);
  }
  60% {
    opacity: 1;
    -webkit-transform: translate3d(-25px, 0, 0);
    transform: translate3d(-25px, 0, 0);
  }
  75% {
    -webkit-transform: translate3d(10px, 0, 0);
    transform: translate3d(10px, 0, 0);
  }
  90% {
    -webkit-transform: translate3d(-5px, 0, 0);
    transform: translate3d(-5px, 0, 0);
  }
  to {
    -webkit-transform: translate3d(0, 0, 0);
    transform: translate3d(0, 0, 0);
  }
}

@keyframes bounceInRight {
  from,
  60%,
  75%,
  90%,
  to {
    -webkit-animation-timing-function: cubic-bezier(0.215, 0.61, 0.355, 1);
    animation-timing-function: cubic-bezier(0.215, 0.61, 0.355, 1);
  }
  from {
    opacity: 0;
    -webkit-transform: translate3d(3000px, 0, 0);
    transform: translate3d(3000px, 0, 0);
  }
  60% {
    opacity: 1;
    -webkit-transform: translate3d(-25px, 0, 0);
    transform: translate3d(-25px, 0, 0);
  }
  75% {
    -webkit-transform: translate3d(10px, 0, 0);
    transform: translate3d(10px, 0, 0);
  }
  90% {
    -webkit-transform: translate3d(-5px, 0, 0);
    transform: translate3d(-5px, 0, 0);
  }
  to {
    -webkit-transform: translate3d(0, 0, 0);
    transform: translate3d(0, 0, 0);
  }
}

.bounceInRight {
  -webkit-animation-name: bounceInRight;
  animation-name: bounceInRight;
}

@-webkit-keyframes fadeIn {
  from {
    opacity: 0;
  }
  to {
    opacity: 1;
  }
}

@keyframes fadeIn {
  from {
    opacity: 0;
  }
  to {
    opacity: 1;
  }
}

.fadeIn {
  -webkit-animation-name: fadeIn;
  animation-name: fadeIn;
}

.animated.infinite {
  -webkit-animation-iteration-count: infinite;
  animation-iteration-count: infinite;
}

