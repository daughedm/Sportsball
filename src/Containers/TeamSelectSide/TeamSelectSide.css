.unordered-list {
  list-style-type: none;
}

.scrollable-menu {
  height: auto;
  max-height: 170px;
  overflow-x: hidden;
  width: 100%;
  border-radius: 4px;
  border: 1px solid #3C3C3A;
  margin-top: 25px;
  background-color: #F4F1E7;
  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.5);
  padding: 0 20px 0 20px;
}

.side-form {
  width: 100%;
  background-color: rgba(60, 60, 58, 0.18);
  border: 1px solid #F4F1E7;
  border-radius: 4px;
  margin: 60px auto;
  padding: 35px;
}

.form-header {
  color: #F26F44;
  font-family: "Noto Sans";
  font-size: 25px;
  font-weight: 700;
  margin-top: 0;
  margin-bottom: 20px;
  letter-spacing: 1px;
}

.drop-down-button {
  display: block;
  width: 100%;
  height: 50px;
  border-radius: 4px;
  border: 1px solid #3C3C3A;
  margin-top: 25px;
  background-color: #F4F1E7;
  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.5);
  padding: 10px;
  font-size: 14px;
  color: #3C3C3A;
}

.submit-button {
  display: block;
  margin-top: 25px;
  width: 100%;
  height: 50px;
  border: none;
  border-radius: 4px;
  background: linear-gradient(0deg, #F26F44 11%, #F98059 52%, #FF8F6B 100%);
  color: #F4F1E7;
  font-size: 16px;
  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.5);
  cursor: pointer;
}

.anchor-toggle {
  font-size: 14px;
  display: block;
  text-decoration: underline;
  color: #F4F1E7;
  font-style: italic;
  padding-top: 12px;
  cursor: pointer;
}

li a {
  text-decoration: none;
  font-size: 16px;
  color: #3C3C3A;
 }

.list-item {
 padding: 15px 0 15px 0;
 height: 50px;
 border-bottom: 1px solid #808080;
}

.unordered-list :hover {
  background-color: #ffe4db;
}

::-webkit-scrollbar {
  width: 8px;
  height: 8px;
}

::-webkit-scrollbar-button {
  width: 0px;
  height: 0px;
}

::-webkit-scrollbar-thumb {
  background: #f26f44;
  border: 2px solid transparent;
  border-radius: 44px;
}

::-webkit-scrollbar-thumb:hover {
  background: #f26f44;
}

::-webkit-scrollbar-thumb:active {
  background: #f26f44;
}

::-webkit-scrollbar-track {
  background: #f4f1e7;
  border: 45px none #ffffff;
  border-radius: 44px;
}

::-webkit-scrollbar-track:hover {
  background: transparent;
}

::-webkit-scrollbar-track:active {
  background: transparent;
}

::-webkit-scrollbar-corner {
  background: transparent;
}