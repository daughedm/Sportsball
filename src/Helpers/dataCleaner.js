import * as prases from './talkingPointPhrases';

const yesterdaysDate = () => {
  let todayTimeStamp = new Date();
  let oneDayTimeStamp = 1000 * 60 * 60 * 24;
  let diff = todayTimeStamp - oneDayTimeStamp;
  let yesterdayDate = new Date(diff);
  let year = yesterdayDate.getFullYear()
  let month = yesterdayDate.getMonth() + 1;
  let day = yesterdayDate.getDate();

  if (month < 10) {
    month = '0' + month;
  }
  if (day < 10) {
    day = '0' + day;
  }

  let yesterdayString = year + '/' + month + '/' + day;
  return yesterdayString;
}

const gameSummaryCleaner = (rawSummaryData) => {
  const cleanedSummary = rawSummaryData.league.games.map(ballGame => {
    return {
      homeTeam: ballGame.game.home.name,
      awayTeam: ballGame.game.away.name,
      homeTeamRuns: ballGame.game.home.runs,
      awayTeamRuns: ballGame.game.away.runs,
      winningTeam: ballGame.game.home.runs > ballGame.game.away.runs ? ballGame.game.home.name : ballGame.game.away.name,
      losingTeam: ballGame.game.home.runs < ballGame.game.away.runs ? ballGame.game.home.name : ballGame.game.away.name
    }
  })
  return cleanedSummary;
}

const boxScoresCleaner = (rawBoxScoresData) => {
  const cleanedBoxScores = rawBoxScoresData.league.games.map(ballgame => {
    return {
      homeTeam: ballgame.game.home.name,
      awayTeam: ballgame.game.away.name,
      homeTeamEvents: ballgame.game.home.events,
      awayTeamEvents: ballgame.game.home.events
    }
  })
  return cleanedBoxScores;
} 

const findGameSummary = (yourTeam) => {
  let singleGame = cleanData.find(ballGame => {
    return ballGame.homeTeam.includes(yourTeam) || ballGame.awayTeam.includes(yourTeam)
  })
  if (singleGame === undefined) {
    singleGame = `The ${yourTeam} didn't play!`
  }
  return singleGame;
}

const findGameBoxScore = (yourTeam) => {
  let singleGame = cleanData.find(ballGame => {
    return ballGame.homeTeam.includes(yourTeam) || ballGame.awayTeam.includes(yourTeam)
  })
  if (singleGame === undefined) {
    singleGame = `The ${yourTeam} didn't play!`
  }
  return singleGame;
}

// const findGame = (yourTeam) => {
//   const singleGame = gameSummaryCleaner.find(ballGame => {
//    return ballGame.homeTeam.includes(yourTeam) || ballGame.awayTeam.includes(yourTeam)
//   })
//   return singleGame;
// }

// const firstTalkingPoint = (yourTeam) => {
//   const teamsGame = findGame(yourTeam);
//   let score = `${teamsGame.homeTeamRuns} - ${teamsGame.awayTeamRuns}`;
//   let randNum;
//   let message;

//   if (teamsGame.winningTeam === yourTeam) {
//     randNum = Math.floor(Math.random() * winningMessages(yourTeam, score).length);
//     console.log(randNum)
//     message = winningMessages(yourTeam, score)[randNum];
//   } else if (teamsGame.winningTeam !== yourTeam && teamsGame.winningTeam !== undefined) {
//     randNum = Math.floor(Math.random() * losingMessages(yourTeam, score).length);
//     message = losingMessages(yourTeam, score)[randNum];
//   } else {
//     message = `The ${yourTeam} didn't play today!`;
//   }
//   return message;
// }

export { 
  yesterdaysDate,
  gameSummaryCleaner,
  boxScoresCleaner
};