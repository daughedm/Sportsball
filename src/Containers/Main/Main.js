import React from 'react'
import './Main.css';
import { connect } from 'react-redux';
import { yesterdaysDate } from '../../Helpers/dataCleaner';
import { winningMessages, losingMessages } from '../../Helpers/talkingPointPhrases';
import { firstTalkingPoint } from '../../Helpers/talkingPointLogic';
import logo from '../../Assets/images/logo_creme.svg';

const Main = (props) => {
  const date = yesterdaysDate();
  const firstPhrase = firstTalkingPoint(props.selectedTeam, props.singleGameSummary)
  
  return (
    <div className="container">
      <div className="left-container">
        <img className="sportsball-logo-side" src={logo}/>
      </div>
      <div className="orange-container">
        <h3 className="talking-point" >YOUR DAILY TALKING POINTS <span className="light"> | </span>   {date} 
        </h3>
        < div className="talking-points-container" >
          <ul className="u-list">
            <li className="point border-bottom">{ firstPhrase }</li>
            <li className="point border-bottom">That double that scored one in the third was clutch!</li>
            <li className="point">Lindor is having a fantastic year, hitting .345</li>     
          </ul>
        </div>
      </div>
    </div>
  )
}

export const mapStateToProps = (state) => ({
  selectedTeam: state.selectedTeam,
  singleGameSummary: state.singleGameSummary,
  singleBoxScore: state.singleBoxScore
})

export default connect(mapStateToProps, null)(Main)