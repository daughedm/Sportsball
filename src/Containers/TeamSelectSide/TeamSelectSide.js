import React, { Component } from 'react';
import PropTypes from 'prop-types';
import { connect } from 'react-redux';
import { findGame, teamStatCleaner } from '../../Helpers/dataCleaner'
import { addSelectedTeamToStore } from '../../Actions/userActions';
import './TeamSelectSide.css';
import { addSingleSummaryToStore, addSingleBoxScoreToStore, addTeamStatsToStore } from '../../Actions';
import { teamStatsFetch } from '../../ApiCall/ApiCall';
import TeamScroll from './../TeamScroll/TeamScroll';

export class TeamSelectSide extends Component {

  render() {
    return ( 
      <form
        className = "side-form"
        type = "submit"
      >
        <h3 className = "form-header" > Switch Teams </h3> 
          <TeamScroll />
      </form>
    )
  }
}

TeamSelectSide.propTypes = {

}

export default TeamSelectSide
