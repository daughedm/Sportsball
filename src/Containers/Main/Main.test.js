import { Main, mapStateToProps } from './Main';
import React from 'react';
import { shallow } from 'enzyme';
import { connect } from 'react-redux';
import { yesterdaysDate } from '../../Helpers/dataCleaner';
import { winningMessages, losingMessages } from '../../Helpers/talkingPointPhrases';
import { firstTalkingPoint, secondTalkingPoint, thirdTalkingPoint } from '../../Helpers/talkingPointLogic';
import TeamSelectSide from '../TeamSelectSide/TeamSelectSide';
import {
  mockCleanSummary,
  mockCleanBoxScores,
  mockCleanTeamStats
} from './../../mockData/mockCleanData';

const mockMath = Object.create(global.Math);
mockMath.random = () => 0;
global.Math = mockMath;

describe('Main',() => {
  let wrapper;
  let mockSelectedTeam;
  let mockSingleGameSummary;
  let mockSingleBoxScore;
  let mockTeamStats;
  
  beforeEach(() => {
    mockSelectedTeam = 'Indians';
    mockSingleGameSummary = mockCleanSummary;
    mockSingleBoxScore = mockCleanBoxScores;
    mockTeamStats= mockCleanTeamStats;
    
    wrapper = shallow(<Main 
      selectedTeam={ mockSelectedTeam }
      singleGameSummary={ mockSingleGameSummary }
      singleBoxScore={ mockSingleBoxScore }
      teamStats={ mockTeamStats }
                      /> );
  });


  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});