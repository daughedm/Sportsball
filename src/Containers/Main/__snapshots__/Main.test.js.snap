// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    selectedTeam="Indians"
    singleBoxScore={
        Object {
            "awayTeam": "Cubs",
            "awayTeamEvents": Array [
              Object {},
              Object {},
            ],
            "homeTeam": "Indians",
            "homeTeamEvents": Array [
              Object {
                "runners": Array [
                  1,
                  2,
                  3,
                ],
              },
              Object {
                "runners": Array [
                  1,
                  2,
                  3,
                ],
              },
            ],
          }
    }
    singleGameSummary={
        Object {
            "awayTeam": "Cubs",
            "awayTeamRuns": 0,
            "homeTeam": "Indians",
            "homeTeamRuns": 100,
            "losingTeam": "Cubs",
            "winningTeam": "Indians",
          }
    }
    teamStats={
        Object {
            "Indians": Array [
              Object {
                "first_name": "Lonnie",
                "id": "7a8633ea-0824-4164-b0b1-e4394da6fc43",
                "jersey_number": "8",
                "last_name": "Chisenhall",
                "position": "OF",
                "preferred_name": "Lonnie",
                "primary_position": "RF",
                "statistics": Object {
                  "fielding": Object {
                    "overall": Object {
                      "a": 0,
                      "assists": Object {
                        "outfield": 0,
                        "total": 0,
                      },
                      "c_wp": 0,
                      "dp": 0,
                      "error": 0,
                      "errors": Object {
                        "fielding": 0,
                        "interference": 0,
                        "throwing": 0,
                        "total": 0,
                      },
                      "fpct": 1,
                      "games": Object {
                        "complete": 0,
                        "finish": 1,
                        "play": 7,
                        "start": 6,
                      },
                      "inn_1": 136,
                      "inn_2": 45.1,
                      "pb": 0,
                      "po": 11,
                      "rf": 1.571,
                      "steal": Object {
                        "caught": 0,
                        "pct": 0,
                        "pickoff": 0,
                        "stolen": 0,
                      },
                      "tc": 11,
                      "tp": 0,
                    },
                    "positions": Array [
                      Object {
                        "a": 0,
                        "assists": Object {
                          "outfield": 0,
                          "total": 0,
                        },
                        "c_wp": 0,
                        "dp": 0,
                        "error": 0,
                        "errors": Object {
                          "fielding": 0,
                          "interference": 0,
                          "throwing": 0,
                          "total": 0,
                        },
                        "fpct": 1,
                        "games": Object {
                          "complete": 0,
                          "finish": 1,
                          "play": 7,
                          "start": 6,
                        },
                        "inn_1": 136,
                        "inn_2": 45.1,
                        "pb": 0,
                        "po": 11,
                        "position": "RF",
                        "rf": 1.571,
                        "steal": Object {
                          "caught": 0,
                          "pct": 0,
                          "pickoff": 0,
                          "stolen": 0,
                        },
                        "tc": 11,
                        "tp": 0,
                      },
                    ],
                  },
                  "hitting": Object {
                    "overall": Object {
                      "ab": 17,
                      "ab_risp": 1,
                      "abhr": 0,
                      "abk": 4.25,
                      "ap": 21,
                      "avg": ".235",
                      "babip": 0.308,
                      "bbk": 0.75,
                      "bbpa": 0.143,
                      "bip": 13,
                      "flyball": 2,
                      "games": Object {
                        "complete": 4,
                        "finish": 1,
                        "play": 7,
                        "start": 6,
                      },
                      "gofo": 1,
                      "groundball": 5,
                      "hit_risp": 0,
                      "iso": 0.059,
                      "linedrive": 5,
                      "lob": 4,
                      "lob_risp_2out": 0,
                      "obp": 0.381,
                      "onbase": Object {
                        "bb": 2,
                        "cycle": 0,
                        "d": 1,
                        "fc": 0,
                        "h": 4,
                        "hbp": 1,
                        "hr": 0,
                        "ibb": 1,
                        "roe": 0,
                        "s": 3,
                        "t": 0,
                        "tb": 5,
                      },
                      "ops": 0.675,
                      "outcome": Object {
                        "ball": 30,
                        "dirtball": 4,
                        "foul": 21,
                        "iball": 0,
                        "klook": 13,
                        "kswing": 9,
                        "ktotal": 22,
                      },
                      "outs": Object {
                        "fidp": 0,
                        "fo": 2,
                        "gidp": 1,
                        "go": 5,
                        "klook": 1,
                        "kswing": 3,
                        "ktotal": 4,
                        "lidp": 0,
                        "lo": 2,
                        "po": 1,
                        "sacfly": 0,
                        "sachit": 0,
                      },
                      "pitch_count": 91,
                      "pitches": Object {
                        "btotal": 35,
                        "count": 91,
                        "ktotal": 56,
                      },
                      "popup": 1,
                      "rbi": 0,
                      "rbi_2out": 0,
                      "runs": Object {
                        "total": 3,
                      },
                      "seca": 0.294,
                      "slg": 0.294,
                      "steal": Object {
                        "caught": 0,
                        "pct": 1,
                        "pickoff": 0,
                        "stolen": 1,
                      },
                      "team_lob": 0,
                      "xbh": 1,
                    },
                  },
                },
              },
            ],
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="left-container"
>
          <img
                    alt="logo"
                    className="sportsball-logo-side animated fadeIn"
                    src="logo_creme-02.svg"
          />
          <h2
                    className="team-selected"
          >
                     
                    Indians
                     
          </h2>
          <TeamSelectSide />
</div>,
        <div
          className="orange-container animated bounceInRight"
>
          <h3
                    className="talking-point"
          >
                    YOUR DAILY TALKING POINTS 
                    <span
                              className="light"
                    >
                               | 
                    </span>
                       
                    2018/06/04
          </h3>
          <div
                    className="talking-points-container"
          >
                    <ul
                              className="u-list"
                    >
                              <li
                                        className="point border-bottom"
                              >
                                        Wow, that game was sick! I canâ€™t believe the Indians won 100 - 0!
                              </li>
                              <li
                                        className="point border-bottom"
                              >
                                        Did you see that that play in the inning that scored 3?
                              </li>
                              <li
                                        className="point"
                              >
                                        Lonnie Chisenhall is having great year hitting .235.
                              </li>
                    </ul>
          </div>
</div>,
      ],
      "className": "container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              alt="logo"
              className="sportsball-logo-side animated fadeIn"
              src="logo_creme-02.svg"
/>,
            <h2
              className="team-selected"
>
               
              Indians
               
</h2>,
            <TeamSelectSide />,
          ],
          "className": "left-container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "logo",
              "className": "sportsball-logo-side animated fadeIn",
              "src": "logo_creme-02.svg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                " ",
                "Indians",
                " ",
              ],
              "className": "team-selected",
            },
            "ref": null,
            "rendered": Array [
              " ",
              "Indians",
              " ",
            ],
            "type": "h2",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3
              className="talking-point"
>
              YOUR DAILY TALKING POINTS 
              <span
                            className="light"
              >
                             | 
              </span>
                 
              2018/06/04
</h3>,
            <div
              className="talking-points-container"
>
              <ul
                            className="u-list"
              >
                            <li
                                          className="point border-bottom"
                            >
                                          Wow, that game was sick! I canâ€™t believe the Indians won 100 - 0!
                            </li>
                            <li
                                          className="point border-bottom"
                            >
                                          Did you see that that play in the inning that scored 3?
                            </li>
                            <li
                                          className="point"
                            >
                                          Lonnie Chisenhall is having great year hitting .235.
                            </li>
              </ul>
</div>,
          ],
          "className": "orange-container animated bounceInRight",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "YOUR DAILY TALKING POINTS ",
                <span
                  className="light"
>
                   | 
</span>,
                "   ",
                "2018/06/04",
              ],
              "className": "talking-point",
            },
            "ref": null,
            "rendered": Array [
              "YOUR DAILY TALKING POINTS ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": " | ",
                  "className": "light",
                },
                "ref": null,
                "rendered": " | ",
                "type": "span",
              },
              "   ",
              "2018/06/04",
            ],
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <ul
                className="u-list"
>
                <li
                                className="point border-bottom"
                >
                                Wow, that game was sick! I canâ€™t believe the Indians won 100 - 0!
                </li>
                <li
                                className="point border-bottom"
                >
                                Did you see that that play in the inning that scored 3?
                </li>
                <li
                                className="point"
                >
                                Lonnie Chisenhall is having great year hitting .235.
                </li>
</ul>,
              "className": "talking-points-container",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <li
                    className="point border-bottom"
>
                    Wow, that game was sick! I canâ€™t believe the Indians won 100 - 0!
</li>,
                  <li
                    className="point border-bottom"
>
                    Did you see that that play in the inning that scored 3?
</li>,
                  <li
                    className="point"
>
                    Lonnie Chisenhall is having great year hitting .235.
</li>,
                ],
                "className": "u-list",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Wow, that game was sick! I canâ€™t believe the Indians won 100 - 0!",
                    "className": "point border-bottom",
                  },
                  "ref": null,
                  "rendered": "Wow, that game was sick! I canâ€™t believe the Indians won 100 - 0!",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Did you see that that play in the inning that scored 3?",
                    "className": "point border-bottom",
                  },
                  "ref": null,
                  "rendered": "Did you see that that play in the inning that scored 3?",
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Lonnie Chisenhall is having great year hitting .235.",
                    "className": "point",
                  },
                  "ref": null,
                  "rendered": "Lonnie Chisenhall is having great year hitting .235.",
                  "type": "li",
                },
              ],
              "type": "ul",
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="left-container"
>
            <img
                        alt="logo"
                        className="sportsball-logo-side animated fadeIn"
                        src="logo_creme-02.svg"
            />
            <h2
                        className="team-selected"
            >
                         
                        Indians
                         
            </h2>
            <TeamSelectSide />
</div>,
          <div
            className="orange-container animated bounceInRight"
>
            <h3
                        className="talking-point"
            >
                        YOUR DAILY TALKING POINTS 
                        <span
                                    className="light"
                        >
                                     | 
                        </span>
                           
                        2018/06/04
            </h3>
            <div
                        className="talking-points-container"
            >
                        <ul
                                    className="u-list"
                        >
                                    <li
                                                className="point border-bottom"
                                    >
                                                Wow, that game was sick! I canâ€™t believe the Indians won 100 - 0!
                                    </li>
                                    <li
                                                className="point border-bottom"
                                    >
                                                Did you see that that play in the inning that scored 3?
                                    </li>
                                    <li
                                                className="point"
                                    >
                                                Lonnie Chisenhall is having great year hitting .235.
                                    </li>
                        </ul>
            </div>
</div>,
        ],
        "className": "container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <img
                alt="logo"
                className="sportsball-logo-side animated fadeIn"
                src="logo_creme-02.svg"
/>,
              <h2
                className="team-selected"
>
                 
                Indians
                 
</h2>,
              <TeamSelectSide />,
            ],
            "className": "left-container",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "logo",
                "className": "sportsball-logo-side animated fadeIn",
                "src": "logo_creme-02.svg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  " ",
                  "Indians",
                  " ",
                ],
                "className": "team-selected",
              },
              "ref": null,
              "rendered": Array [
                " ",
                "Indians",
                " ",
              ],
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3
                className="talking-point"
>
                YOUR DAILY TALKING POINTS 
                <span
                                className="light"
                >
                                 | 
                </span>
                   
                2018/06/04
</h3>,
              <div
                className="talking-points-container"
>
                <ul
                                className="u-list"
                >
                                <li
                                                className="point border-bottom"
                                >
                                                Wow, that game was sick! I canâ€™t believe the Indians won 100 - 0!
                                </li>
                                <li
                                                className="point border-bottom"
                                >
                                                Did you see that that play in the inning that scored 3?
                                </li>
                                <li
                                                className="point"
                                >
                                                Lonnie Chisenhall is having great year hitting .235.
                                </li>
                </ul>
</div>,
            ],
            "className": "orange-container animated bounceInRight",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "YOUR DAILY TALKING POINTS ",
                  <span
                    className="light"
>
                     | 
</span>,
                  "   ",
                  "2018/06/04",
                ],
                "className": "talking-point",
              },
              "ref": null,
              "rendered": Array [
                "YOUR DAILY TALKING POINTS ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": " | ",
                    "className": "light",
                  },
                  "ref": null,
                  "rendered": " | ",
                  "type": "span",
                },
                "   ",
                "2018/06/04",
              ],
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <ul
                  className="u-list"
>
                  <li
                                    className="point border-bottom"
                  >
                                    Wow, that game was sick! I canâ€™t believe the Indians won 100 - 0!
                  </li>
                  <li
                                    className="point border-bottom"
                  >
                                    Did you see that that play in the inning that scored 3?
                  </li>
                  <li
                                    className="point"
                  >
                                    Lonnie Chisenhall is having great year hitting .235.
                  </li>
</ul>,
                "className": "talking-points-container",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <li
                      className="point border-bottom"
>
                      Wow, that game was sick! I canâ€™t believe the Indians won 100 - 0!
</li>,
                    <li
                      className="point border-bottom"
>
                      Did you see that that play in the inning that scored 3?
</li>,
                    <li
                      className="point"
>
                      Lonnie Chisenhall is having great year hitting .235.
</li>,
                  ],
                  "className": "u-list",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Wow, that game was sick! I canâ€™t believe the Indians won 100 - 0!",
                      "className": "point border-bottom",
                    },
                    "ref": null,
                    "rendered": "Wow, that game was sick! I canâ€™t believe the Indians won 100 - 0!",
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Did you see that that play in the inning that scored 3?",
                      "className": "point border-bottom",
                    },
                    "ref": null,
                    "rendered": "Did you see that that play in the inning that scored 3?",
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Lonnie Chisenhall is having great year hitting .235.",
                      "className": "point",
                    },
                    "ref": null,
                    "rendered": "Lonnie Chisenhall is having great year hitting .235.",
                    "type": "li",
                  },
                ],
                "type": "ul",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
